include/cantera/oneD/OneDim.h

```diff
@@ -45,7 +45,8 @@ class OneDim
     //!     superseded by addDomain() using shared_ptr<Domain1D>
     void addDomain(Domain1D* d);
 
-    //! Return a reference to the Jacobian evaluator.
+    //! Return a reference to the Jacobian evaluator of an OneDim object.
+    //! @ingroup derivGroup
```

Comment:
I see what you're trying to do by adding the class name to the docstring, but the way member functions are shown in Doxygen's "modules" leaves a fair bit to be desired, as I'm sure you've noticed. While I think the two groups touched on in this PR are okay, I'd suggest not going any further in the direction of trying to populate such groups, if you hadn't already come to that conclusion.

Reply:
I tried to collect some spots where Jacobians are evaluated, as they are relevant to the new `derivGroup` module. I indeed tried to mitigate the lacking presentation. I have no intention to make this more wide-spread.

Reply:
`breathe` may eventually provide us with an opportunity to adjust the presentation, so I think this is fine for now.